examples/rrdtool/BUILD

```diff
@@ -0,0 +1,57 @@
+load("@rules_foreign_cc//foreign_cc:defs.bzl", "configure_make")
+
+filegroup(
+    name = "all_srcs",
+    srcs = glob(["**"]),
+)
+
+# TODO(sp): Use -fdata-sections -ffunction-sections + -Wl,--gc-sections (plus
+# perhaps -Wl,--print-gc-sections)
+configure_make(
+    name = "rrdtool",
+    args = ["-j6"],
+    configure_options = [
+        "--disable-docs",
+        "--disable-examples",
+        "--disable-libbdi",
+        "--disable-libwrap",
+        "--disable-lua",
+        "--disable-perl",
+        "--disable-python",
+        "--disable-ruby",
+        "--disable-rrdcaches",
+        "--disable-rrdcgi",
+        "--disable-tcl",
+        "--with-systemdsystemunitdir=no",
+        # https://github.com/bazelbuild/rules_foreign_cc/issues/239
+        "CFLAGS='-Dredacted=\"redacted\"'",
```

Comment:
I think that doesn't append but replaces the `CFLAGS`.
However, Bazel passes the include directories to the compiler with `CFLAGS=-isystem ...`. Replacing `CFLAGS` means no header is found and configure fails.